packages/astro/src/@types/astro.ts

```diff
@@ -1375,6 +1375,91 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		devOverlay?: boolean;
+
+		// TODO review with docs team before merging to `main`
+		/**
+		 * @docs
+		 * @name experimental.i18n
+		 * @type {object}
+		 * @version 3.5.0
+		 * @type {object}
+		 * @description
+		 *
+		 * Configures experimental i18n routing and allows you to specify some customization options.
+		 */
+		i18n?: {
+			/**
+			 * @docs
+			 * @name experimental.i18n.defaultLocale
+			 * @type {string}
+			 * @version 3.5.0
+			 * @description
+			 *
+			 * The default locale of your website/application
+			 */
+			defaultLocale: string;
+			/**
+			 * @docs
+			 * @name experimental.i18n.locales
+			 * @type {string[]}
+			 * @version 3.5.0
+			 * @description
+			 *
+			 * A list of all locales supported by the website. e.g. `['en', 'fr', 'es']` This list should/should not include the `defaultLocale`.
```

Comment:
```suggestion
			 * A list of all locales supported by the website. e.g. `['en', 'fr', 'es']` This list should also include the `defaultLocale`.
```
Just calling out that we need to pick one!  😄 

I'm assuming from the bug bash that `defaultLocale` SHOULD be included in this list, so that's the edit I'm suggesting. If not, fix it the other way!

I will also note that once we define a system/syntax, we should mention that same one both here and in `defaultLocale`.

QUESTION: Is it true that right now you don't really *care* what syntax is used, as long as the folder structure matches what's listed here?

E.g. You could make up a language `dqwx` and as long as you have files in `src/pages/dqwx/` the routing system functions properly?

In that case, I'd suggest wording like:

> No particular language format or syntax is enforced, but your folder structure must match exactly the `locales` in the list.